src/openrct2-ui/CursorRepository.cpp

```diff
@@ -125,17 +124,13 @@ void CursorRepository::GenerateScaledCursorSetHolder(uint8 scale)
 {
     if (_scaledCursors.find(scale) == _scaledCursors.end())
     {
-        std::function<SDL_Cursor *(CURSOR_ID)> cursorGenerator = [this, scale](CURSOR_ID cursorId)
-        {
+        std::function<SDL_Cursor*(CURSOR_ID)> cursorGenerator = [this, scale](CURSOR_ID cursorId) {
             switch (cursorId)
             {
                 // We can't scale the system cursors, but they should be appropriately scaled anyway
-                case CURSOR_ARROW:
-                    return SDL_CreateSystemCursor(SDL_SYSTEM_CURSOR_ARROW);
-                case CURSOR_HAND_POINT:
-                    return SDL_CreateSystemCursor(SDL_SYSTEM_CURSOR_HAND);
-                default:
-                    return this->Create(GetCursorData(cursorId), scale);
+                case CURSOR_ARROW: return SDL_CreateSystemCursor(SDL_SYSTEM_CURSOR_ARROW);
```

Comment:
Clang-format should not put those on one line.

Reply:
Looks like `AllowShortCaseLabelsOnASingleLine` can fix this.